Package.swift

```diff
@@ -0,0 +1,29 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "FTLinearActivityIndicator",
+    platforms: [
+        .iOS(.v8)
+    ],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "FTLinearActivityIndicator",
+            targets: ["FTLinearActivityIndicator"])
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(
+            name: "FTLinearActivityIndicator",
+            dependencies: [])
```

Comment:
You can specify a different directory like so: 
```swift
.target(
    ...
    path: "FTLinearActivityIndicator"
    ...
)
```

Reply:
I did this because I want to conform to SPM standard, since it isn't a breaking change.
If you want I can still roll back to your desired path specification

Reply:
I think @futuretap should decide what he wants.